README.md

```diff
@@ -716,6 +716,19 @@ const edd = easydropdown(selectElement);
 edd.refresh();
 ```
 
+### validate()
+
+`.validate()`
+
+Validates the instance and returns a boolean whether validation succeeded. Helpful if you need to validate without hitting the form submit button. 
+
+```js
+
+const edd = easydropdown(selectElement);
+
+edd.validate();
```

Comment:
Please extend the example to demonstrate how we can use the returned boolean, e.g:

```js
const isValid = edd.validate();

console.log(isValid); // false
```